### DISCUSSION TITLE
The school you selected does not appear to have a campus location in your country

### DISCUSSION THREAD

**Discussion by mhbahmani** [community/community#41131] [GitHub Education]
Hi,
I'm trying to get a student pack, but github keeps saying "The school you selected does not appear to have a campus location in your country". How can I prove that there is a campus in my location? :)
My location in my profile is updated and my primary email is my university email.

**Comment by samus-aran** [comment]
Hi @mhbahmani 👋 

Sorry you're having difficulties applying for Global Campus, let's see if we can assist.

1. Ensure you have allowed your browser to use your location - this will help calculate nearby campuses more accurately.
2. Ensure text is illegible on the submitted evidence photo file - if your evidence contains non-english text try translating it in English and take a photo of the translation next to the evidence.

Let us know how you get on 👍
